input-output-hk/daedalus
# [DDW-92] Improve error messages on connecting screen

This PR improves the handling of `CERT_NOT_YET_VALID` error that is sent by the api if the Daedalus tls cert is invalid (e.g: when the system time was changed)

## Todo
- [x] Japenese translation for the error message

## How to Test
it's easiest if you run Daedalus against `staging` network and set your system time back ~20 minutes. You should see the connecting screen with the following message:

![screenshot 2019-03-11 um 13 46 44](https://user-images.githubusercontent.com/172414/54137121-56a67280-441d-11e9-906b-09e6ba461f34.png)

When you correct the system time it should recover after ~10 seconds.

### Basics


### Code Quality

### Testing

### After Review:
